db/schema.rb

```diff
@@ -259,6 +269,16 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "news_article", force: :cascade do |t|
+    t.string "headline"
+    t.datetime "effective_date"
+    t.string "body"
+    t.integer "priority"
+    t.datetime "expiration_date"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
```

Comment:
What's up with this? This doesn't look like it's related to the rest of this PR, and it looks like a duplicate of the `news_articles` table created below, but this one is singular, not plural.